app/services/support_interface/candidate_journey_tracker.rb

```diff
@@ -141,9 +151,16 @@ def received_references
       @received_references ||= all_references.select(&:feedback_provided?)
     end
 
-    def received_reference_times
+    def received_references_timings
       @received_reference_times ||=
```

Comment:
Worth updating the instance var name as well for consistency.

Reply:
Yea good point 👍